README.md

```diff
@@ -8,6 +8,38 @@
 
 Installation and setup guide can be found here: [Setup guide](./docs/SETUP.md).
 
+## Usage with `MaterialPaletteProvider` and `withMaterialPalette`
```

Comment:
If you're using named import it should tell you exactly what it is and in React devtools you would see a component named `MaterialPaletteProvider`, not `PaletteProivder`, so it's more clear what exactly the component is.